integration/src/test/scala/org/knora/webapi/testservices/TestClientService.scala

```diff
@@ -155,52 +160,30 @@ final case class TestClientService(
     } yield json
 
   /**
-   * Uploads a file to the IIIF Service's "upload" route and returns the information in Sipi's response.
-   * The upload creates a multipart/form-data request which can contain multiple files.
+   * Uploads a file to the Ingest service's "/projects/$shortcode/assets/ingest/$filename" route.
```

Comment:
```suggestion
   * Uploads a file to the DSP-INGEST service's "/projects/$shortcode/assets/ingest/$filename" route.
```

Reply:
If we use that everywhere, we have a lot of places to fix. 😁